src/style.css

```diff
@@ -1,13 +1,99 @@
-@import "./styles/normalize.css";
-@import "./styles/image.css";
-@import "./styles/prices.css";
-@import "./styles/button.css";
+@import "./styles/links.css";
```

Comment:
you should put all styles in 'styles' folder, then create files that includes all block elements styles like card__*** and stars__*** and after it import all files onto index.css and put index.css to your html